### PR TITLE
Another fix for abandoning claims

### PR DESCRIPTION
**Pull request by hackrush01**

### CODE REVIEW

File: src/renderer/redux/actions/app.js [hunk 1]
```diff
@@ -222,6 +222,7 @@ export function doDaemonReady() {
     dispatch(doBalanceSubscribe());
     dispatch(doFetchFileInfosAndPublishedClaims());
     dispatch(doFetchRewardedContent());
+    dispatch(doFetchTransactions(false));
```
Comment (kauffj) [inline src/renderer/redux/actions/app.js]: Why this change?

Comment (hackrush01) [reply src/renderer/redux/actions/app.js]: Without fetch tip info the command is and offline command and it populates the field `claim_id` and `claim_name` field required for abandon's reducers. I put it there to have a less costly API call.
(Is this an example of premature optimization{and thereby bad coding practice}?).
